app/Support/BackupManager.php

```
@@ -0,0 +1,620 @@
+        }
+        foreach (array_diff($files, ['.', '..']) as $file) {
+            $path = $dir . '/' . $file;
+            if (is_dir($path)) {
+                $this->deleteDirectory($path);
+            } else {
+                // nosemgrep: php.lang.security.unlink-use.unlink-use -- recursive delete within an already-validated backup directory
+                @unlink($path);
+            }
+        }
+        @rmdir($dir);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 593. (undefined)

(ErrorControlOperator)